### DISCUSSION TITLE
Should FullWindowOverlay always have accessibilityviewismodal as true?

### DISCUSSION THREAD

**Discussion by wahidshafique** [software-mansion/react-native-screens#2029] [Ideas]
This was prompted by a bug that I encountered once I updated to the latest version of rns. Previously I had a fullscreenoverlay that was persistent, used for toast notifications. However after [this](https://github.com/software-mansion/react-native-screens/pull/1842) change was merged in, I could not interact with any elements underneath the overlay using the ios accessibility inspector.

I'm wondering whether [accessibilityviewismodal](https://developer.apple.com/documentation/objectivec/nsobject/1615089-accessibilityviewismodal) should be an configurable prop, as the overlay doesn't necessarily mean I want the screen reader to be captured.

What does everyone else think?

**Comment by tboba** [comment]
Hey @wahidshafique, thanks for making this discussion!
Yeah, I think it's a good idea to make the content underneath FullWindowOverlay interactive. I'll leave this topic to the ideas category.

**Comment by bitensam** [comment]
Hi, I have encountered the same problem. Any chance to make those a11y props configurable ?

**Comment by mzablockifm** [comment]
This might become a bigger problem because European Accessibility Act comes into effect in June 2025.

**Comment by kkafar** [comment]
@bitensam @mzablockifm 

Hey, I haven't answered here, but you can customise this behaviour by `unstable_accessibilityContainerViewIsModal`. See this PR:

* https://github.com/software-mansion/react-native-screens/pull/2854

for details.
